assets/support.rackspace.com/src/css/_sass/_headerstyle.scss

```diff
@@ -443,8 +443,8 @@ a {
       }
 
       &.rax-logo {
-        background-image: url("https://b62acf5a1390c5b084e0-4fd83007cb8815e8cf99a7a92e9d6a73.ssl.cf2.rackcdn.com/logo-rackspace.png");
-        background-image: none, url("https://b62acf5a1390c5b084e0-4fd83007cb8815e8cf99a7a92e9d6a73.ssl.cf2.rackcdn.com/logo-rackspace.svg");
+        background-image: url("../../../../src/img/rackspace-logo.png");
+        background-image: none, url("../../../../src/img/rackspace-logo.svg");
```

Comment:
Shouldn't the URL be `assets/src` instead of `../../../../src`

Reply:
@DonNateR It might work. `headerstyle.scss` is in the `assets` dir, so we need to figure out the proper number of `../` before `/src`. One way to check the path is to run this in the local Deconst env and see the result.

Reply:
I'll try to test it in mine and see what happens.

Reply:
Shall I update the file to have assets/src/ instead of all the ../s?

Reply:
As written, it doesn't seem to find the image
![image](https://user-images.githubusercontent.com/730187/43961325-3988a600-9c7a-11e8-8643-e0409eef0599.png)

Reply:
@robb-romans: Can you try replacing the file path with `assets/src` instead of `../../../../src`?

Reply:
I found this: https://stackoverflow.com/questions/20047364/how-to-give-the-background-image-path-in-css/20047411#20047411
It suggests that you need to path it like I did (unless i miscounted parent directories.
the image is in: assets/src/img directory
the css is in: assets/support.rackspace.com/src/css/_sass directory
since teh css is referencing the image, shouldn't it need to go up four levels and then back down?

Reply:
Do we need to abandon the idea of using the local file and go back to figuring how to update the cloudcdn image?